pkg/controller/perconaservermongodb/mgo.go

```diff
@@ -40,6 +40,9 @@ func (r *ReconcilePerconaServerMongoDB) reconcileCluster(ctx context.Context, cr
 		return api.AppStateReady, nil
 	}
 
+	// Primary with a Secondary and an Arbiter (PSA)
+	unsafePsa := cr.Spec.UnsafeConf && replset.Arbiter.Enabled && replset.Arbiter.Size == 1 && !replset.NonVoting.Enabled && replset.Size == 2
```

Comment:
please use `unsafePSA` as variable name, initialisms should be all caps: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Oh I'm sorry, didn't see that correctly. I'll change it.